README.md

```diff
@@ -29,46 +29,73 @@ Our package fills a gap in the python ecosystem by being marketed specifically t
 
 ## Installation
 
+### For User
+
+To install the package, run the command pip install quanteda
```

Comment:
I think it would be useful if this was a copy-able command like the others below

```suggestion
To install the package, run the following command from the terminal
```bash
pip install quanteda
```

```

Reply:
totally agreed!  Thanks for catching it.